migrations/versions/17de7a1bad16_.py

```diff
@@ -0,0 +1,327 @@
+"""empty message
+
+Revision ID: 17de7a1bad16
+Revises: bbf13d065442
+Create Date: 2023-04-21 00:18:06.044764
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '17de7a1bad16'
+down_revision = 'bbf13d065442'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    # op.drop_table('bitcoin_prices')
```

Comment:
Can we remove these commented out lines from the migration?

Reply:
Sure